samples-final/8.AspNetCore-LUIS-Bot/Startup.cs

```diff
@@ -65,14 +54,71 @@ public void ConfigureServices(IServiceCollection services)
                 //      https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/
 
                 // IStorage dataStore = new Microsoft.Bot.Builder.Azure.AzureTableStorage("AzureTablesConnectionString", "TableName");
-                // IStorage dataStore = new Microsoft.Bot.Builder.Azure.AzureBlobStorage("AzureBlobConnectionString", "containerName");
+                //IStorage dataStore = new Microsoft.Bot.Builder.Azure.AzureBlobStorage("AzureBlobConnectionString", "containerName");
+                IStorage dataStore = new MemoryStorage();
 
-                options.Middleware.Add(new ConversationState<Dictionary<string, object>>(dataStore));
-                options.Middleware.Add(new UserState<UserState>(dataStore));
+                // *NEW* CREATE NEW CONVERSATION STATE                
```

Comment:
We are doing user state here but the comment says conversation state?

Reply:
Fixed